public/js/pimcore/layout/toolbar.js

```diff
@@ -942,6 +942,15 @@ pimcore.layout.toolbar = Class.create({
          // building the html markup for the main navigation
          pimcore.helpers.buildMainNavigationMarkup(menu);
 
+         // Full menu can be checked here
+         const postMenuBuild = new CustomEvent(pimcore.events.postMenuBuild, {
```

Comment:
IMHO this is too "early" to be called `postMenuBuild`, sounds more something that should be happening after all the submenus are built
https://github.com/pimcore/admin-ui-classic-bundle/pull/387/files#diff-b804c1370a5785827a2dbd64fba0c17e9f5f66526531fe54116207175c8b40c2R958

I'd rename as `postMainNavigationBuild`, `postRootMenuBuild` or so.

Reply:
Moved it to be the very last step in the function. So it is truly post menu build